appengine/parse-server/README.md

```diff
@@ -0,0 +1,29 @@
+# Parse-server sample for Google App Engine
+
+This sample demonstrates deploying a [Parse-server](https://github.com/ParsePlatform/parse-server)
+app to [Google App Engine Managed VMs](https://cloud.google.com/appengine).
+
+## Setup
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project.
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+1. Setup a MongoDB server:
```

Comment:
Can you say something like "here are two options"? I can imagine someone doing both.